strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/artifact/coordinates/PathNupkg.java

```diff
@@ -64,22 +78,31 @@ public Hash getHash()
             return null;
         }
         String checkSumStr = checkSumContents.iterator().next();
-        Hash result = new Hash(Base64.getDecoder().decode(checkSumStr.getBytes("UTF-8")));
-        return result;
-    }
+        return new Hash(Base64.getDecoder().decode(checkSumStr.getBytes("UTF-8")));
+	}
 
     @Override
     public NuspecFile getNuspecFile()
         throws NugetFormatException
     {
-        NugetArtifactCoordinates artifactCoordinates = readArtifactCooridnates();
+        return nuspecFile;
+    }
+
+	private NuspecFile createNuspecFile() throws NugetFormatException {
+		NugetArtifactCoordinates artifactCoordinates = readArtifactCooridnates();
         RepositoryPath parentPath = path.getParent();
```

Comment:
Cosmetic: It seems this method isn't indented as per the convention.

Reply:
fixed